lib/iris/etc/pp_save_rules.txt

```diff
@@ -761,14 +761,14 @@ THEN
 
 #MDI
 IF
-    cm.fill_value is not None
+    isinstance(cm.data, ma.core.MaskedArray)
```

Comment:
@lbdreyer I believe that PR pretty much got removed as a result of the merge back of the original dask feature branch into master